src/EA.Weee.Web/Areas/AATFReturn/Controllers/ObligatedSentOnController.cs

```diff
@@ -0,0 +1,96 @@
+﻿namespace EA.Weee.Web.Areas.AatfReturn.Controllers
+{
+    using EA.Prsd.Core.Mapper;
+    using EA.Weee.Api.Client;
+    using EA.Weee.Requests.AatfReturn;
+    using EA.Weee.Requests.AatfReturn.Obligated;
+    using EA.Weee.Web.Areas.AatfReturn.Mappings.ToViewModel;
+    using EA.Weee.Web.Areas.AatfReturn.Requests;
+    using EA.Weee.Web.Areas.AatfReturn.ViewModels;
+    using EA.Weee.Web.Constant;
+    using EA.Weee.Web.Infrastructure;
+    using EA.Weee.Web.Services;
+    using EA.Weee.Web.Services.Caching;
+    using System;
+    using System.Threading.Tasks;
+    using System.Web.Mvc;
+
+    public class ObligatedSentOnController : AatfReturnBaseController
+    {
+        private readonly Func<IWeeeClient> apiClient;
+        private readonly IObligatedSentOnWeeeRequestCreator requestCreator;
+        private readonly BreadcrumbService breadcrumb;
+        private readonly IWeeeCache cache;
+        private readonly IMap<ReturnToObligatedViewModelMapTransfer, ObligatedViewModel> mapper;
+
+        public ObligatedSentOnController(IWeeeCache cache,
+            BreadcrumbService breadcrumb,
+            Func<IWeeeClient> apiClient,
+            IMap<ReturnToObligatedViewModelMapTransfer, ObligatedViewModel> mapper,
+            IObligatedSentOnWeeeRequestCreator requestCreator)
+        {
+            this.apiClient = apiClient;
+            this.breadcrumb = breadcrumb;
+            this.cache = cache;
+            this.mapper = mapper;
+            this.requestCreator = requestCreator;
+        }
+
+        [HttpGet]
+        public virtual async Task<ActionResult> Index(Guid returnId, Guid organisationId, Guid weeeSentOnId, Guid aatfId)
+        {
+            using (var client = apiClient())
+            {
+                var operatorRequest = new GetSentOnOperatorSite(weeeSentOnId);
+
+                var weeeSentOnSite = await client.SendAsync(User.GetAccessToken(), new GetSentOnAatfSite(weeeSentOnId));
+
+                var weeeSentOnOperator = await client.SendAsync(User.GetAccessToken(), operatorRequest);
```

Comment:
I wonder if we can just pass this operator name on the query string (from the previous screen) rather than having to do an API call?